.vscode/settings.json

```
@@ -0,0 +1,10 @@
+{
+  "editor.formatOnSave": true,
+  "solidity.packageDefaultDependenciesContractsDirectory": "src/contracts/",
+  "solidity.packageDefaultDependenciesDirectory": "lib",
+  "[solidity]": {
+    "editor.defaultFormatter": "JuanBlanco.solidity"
+  },
+  "solidity.formatter": "forge",
```

I'd include this information only when it's actually enforced in the repo.
With autoformatting on, this means you can't save without making undesired changes to the file.
Also, we don't use that import path yet, so we're breaking our tooling.

```suggestion
  "[solidity]": {
    "editor.defaultFormatter": "JuanBlanco.solidity"
  },
```